### PR TITLE
Refreshed text for activity log entries

### PR DESCRIPTION
**Pull request by techsmyth**
Added parentDisplayName to activityLogEntry views.

### CODE REVIEW

File: src/domain/shared/components/ActivityLog/views/ActivityLogCanvasCreatedView.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export interface ActivityLogCanvasCreatedViewProps extends ActivityLogViewProps
 
 export const ActivityLogCanvasCreatedView: FC<ActivityLogCanvasCreatedViewProps> = props => {
   const { t } = useTranslation();
-  const action = t('components.activity-log-view.actions.canvas-created');
+  const action = t('components.activity-log-view.actions.canvas-created', { parentDisplayName: 'calloutDisplayName' });
```
Comment (alstojanovic) [inline src/domain/shared/components/ActivityLog/views/ActivityLogCanvasCreatedView.tsx:10]: Should be `{ parentDisplayName: props.parentDisplayName }`.

File: src/domain/shared/components/ActivityLog/views/ActivityLogCardCommentCreatedView.tsx [hunk 2]
```diff
@@ -7,7 +7,9 @@ export interface ActivityLogCardCommentCreatedViewProps extends ActivityLogViewP
 
 export const ActivityCardCommentCreatedView: FC<ActivityLogCardCommentCreatedViewProps> = props => {
   const { t } = useTranslation();
-  const action = t('components.activity-log-view.actions.card-comment-created');
+  const action = t('components.activity-log-view.actions.card-comment-created', {
+    parentdisplayName: 'cardDisplayName',
```
Comment (alstojanovic) [inline src/domain/shared/components/ActivityLog/views/ActivityLogCardCommentCreatedView.tsx:11]: Should be `{ parentDisplayName: props.parentDisplayName }`

File: src/domain/shared/components/ActivityLog/views/ActivityLogDiscussionCommentCreatedView.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ export interface ActivityLogDiscussionCommentCreatedViewProps extends ActivityLo
 
 export const ActivityLogDiscussionCommentCreatedView: FC<ActivityLogDiscussionCommentCreatedViewProps> = props => {
   const { t } = useTranslation();
-  const action = t('components.activity-log-view.actions.discussion-comment-created');
+  const action = t('components.activity-log-view.actions.discussion-comment-created', { displayName: 'displayName' });
```
Comment (alstojanovic) [inline src/domain/shared/components/ActivityLog/views/ActivityLogDiscussionCommentCreatedView.tsx:10]: Should be `{ parentDisplayName: props.parentDisplayName }`

File: src/domain/shared/components/ActivityLog/views/ActivityLogMemberJoinedView.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ export interface ActivityLogmemberJoinedViewProps extends ActivityLogViewProps {
 
 export const ActivityLogMemberJoinedView: FC<ActivityLogmemberJoinedViewProps> = props => {
   const { t } = useTranslation();
-  const action = t('components.activity-log-view.actions.member-joined');
+  const action = t('components.activity-log-view.actions.member-joined', { communityType: 'communityType' });
```
Comment (alstojanovic) [inline src/domain/shared/components/ActivityLog/views/ActivityLogMemberJoinedView.tsx:10]: Should be `{ parentDisplayName: props.parentDisplayName }`

**Review by alstojanovic — CHANGES_REQUESTED** [review]
Works for adding a Card to a Card Callout but doesn't work for commenting on a Discussion. I tried applying the changes I suggest in the review but they are not enough to make it work.
It looks like the `parentDisplayName` is not set properly.

File: src/domain/shared/components/ActivityLog/views/ActivityLogCanvasCreatedView.tsx (hunk 1 above)
Comment (techsmyth) [reply src/domain/shared/components/ActivityLog/views/ActivityLogCanvasCreatedView.tsx]: Fixed

File: src/domain/shared/components/ActivityLog/views/ActivityLogDiscussionCommentCreatedView.tsx (hunk 3 above)
Comment (techsmyth) [reply src/domain/shared/components/ActivityLog/views/ActivityLogDiscussionCommentCreatedView.tsx]: Fixed

File: src/domain/shared/components/ActivityLog/views/ActivityLogCardCommentCreatedView.tsx (hunk 2 above)
Comment (techsmyth) [reply src/domain/shared/components/ActivityLog/views/ActivityLogCardCommentCreatedView.tsx]: Fixed

**Comment by techsmyth** [comment]
> Works for adding a Card to a Card Callout but doesn't work for commenting on a Discussion. I tried applying the changes I suggest in the review but they are not enough to make it work. It looks like the `parentDisplayName` is not set properly.

Sorry for the silly mistakes, think I missed something with what I ended up checking in. My bad.

The issue re discussion callout comments was not easy to track down so I will raise it in a separate issue. It looks to me that this whole section is actually in need of a design review - it feels too complex for what it is.

**Review by alstojanovic — APPROVED** [review]

**Pull request merged**